packages/kbn-shared-ux-components/src/page_template/solution_nav/solution_nav.tsx

```diff
@@ -58,6 +58,10 @@ export type KibanaPageTemplateSolutionNavProps = Omit<
    */
   isOpenOnDesktop?: boolean;
   onCollapse?: () => void;
+  /*
+   * Force the open state on desktop
+   */
+  isForceOpenOnDesktop?: boolean;
```

Comment:
@cchaos I was trying to achieve a state where the side nav is expanded but no "collapse" button is displayed, that is currently not possible if only `isOpenOnDesktop` prop. 
I'm planning for follow up work (when https://github.com/elastic/eui/pull/5935 is available in Kibana) to rename the prop to `isForceOpen` and it will affect all screens. I think this pattern is kinda similar to for example `forceState` of the EUI accordion. WDYT?